runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -313,6 +345,64 @@ decl_storage! {
 
         /// Grace period for the 'spending' proposal
         pub SpendingProposalGracePeriod get(spending_proposal_grace_period) config(): T::BlockNumber;
+
```

Comment:
Let's keep this for now, but this debt is enumerated here

- https://github.com/Joystream/joystream/issues/862
- https://github.com/Joystream/joystream/issues/863
- https://github.com/Joystream/joystream/issues/864